### PR TITLE
TWIRF-1

### PR DESCRIPTION
**Pull request by dns2utf8**

### CODE REVIEW

File: _posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+[Our CFP](http://cfp.rustfest.eu/) is open again! Please consider submitting! Not everyone will be selected as a speaker, but taking time to think of a topic and put together a proposal can cause other, even unrelated, epiphanies!
+
+Your talk could be about anything (even tangentially) Rust related. It could be about code, about stories from the trenches (of debugging), theory, or anything in between! Please check the link to learn more, even if the idea of speaking is a bit intimidating. You can do it!
+
+<iframe width="640" height="360" src="https://www.youtube.com/embed/videoseries?list=PL85XCvVPmGQj9mqbJizw-zi-EhcpS5jTP" frameborder="0" allowfullscreen></iframe>
+
+
+# Supporter tickets and Diversity fund donations
+
+Like last year we will support underrepresented groups like mothers or students by arranging travel, accommodation and ticket for them.
```
Comment (Hoverbear) [inline _posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md]: I don't think we need to give examples.

File: _posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+authors:
+  - dns2utf8
+  - hoverbear
+---
+
+With the next iteration of RustFest coming it means it's time for new `TwirfIterator`! Welcome to our first `Some(Post)`.
+
+# CFP
+
+[Our CFP](http://cfp.rustfest.eu/) is open again! Please consider submitting! Not everyone will be selected as a speaker, but taking time to think of a topic and put together a proposal can cause other, even unrelated, epiphanies!
```
Comment (tglman) [inline _posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md]: shall we use `https://cfp.rustfest.eu/` here it does exist and seeps to work

Comment (dns2utf8) [reply _posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md]: fixed, thank you

File: _posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md (hunk 1 above)
Comment (dns2utf8) [reply _posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md]: :+1:

**Review by Hoverbear — APPROVED** [review]

**Comment by Hoverbear** [comment]
(This was merged, @dns2utf8 just decided to do it differently or whatever)

**Pull request closed without merging**